app.py

```diff
@@ -14,6 +13,30 @@
 async def start(request):
     return text("pong")
 
+@app.route("/matrix", methods=["GET", "POST"])
+@requires_db
+async def matrix(db, request):
+    if request.method == "GET":
+        ranking_matrices = [
+            {
+                "kind": matrix.kind,
+                "build_options": matrix.build_options,
+                "ranking_matrix_path": matrix.ranking_matrix_path
+            }
+            for matrix in list_ranking_matrix(db, count=10)
+        ]
+        return json(ranking_matrices)
+    elif request.method == "POST":
+        return json({"ping": "pong"})
+
+def list_ranking_matrix(db, count):
```

Comment:
We can use plurals here, `list_ranking_matrices_from_db`.... we'll get rid of the `from_db` part as the design evolve.